challenges/objects-arrays.js

```diff
@@ -55,11 +82,30 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for (let i = 0; i < graduates.length; i++){
+  contactInfo.push(graduates[i].first_name +' '+ graduates[i].email);
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+  
+for (let i = 0; i < graduates.length; i++){
```

Comment:
This doesn't quite accomplish the goal in the request. I think that you were missing the `search()` array method. 
Here is an example of the correct way to solve this:
```
const uni = [];
for(let i = 0; i <graduates.length ;i++){
  if(graduates[i].university.search('Uni') !== -1)
    uni.push(graduates[i].university)
}
console.log(uni);
```
Please review it and let me know if you have any questions!